### PR TITLE
added views

### PR DESCRIPTION
**Pull request by Mithi-code**
### Blog app - Views

- Use preprocessed HTML file with embedded Ruby code.
- Use layouts and templates for shared content.

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,16 +1,31 @@
 <!DOCTYPE html>
```
Comment (Meltrust) [inline app/views/layouts/application.html.erb:1]: - [ ] In order to be able to navigate all the project views without having to resort to the address bar of the browser, please, kindly add links:

- Clicking on each user takes you to their show page as shown in the user show wireframe.
- "See all Post" button opens the user's posts index as displayed on the posts wireframe.
- Clicking on a post on the user's posts index goes to the post details page and displays the post with all comments listed as shown on the post details wireframe.

## Explanation

We need to be able to navigate the project, without resorting to the address bar.

## Hints

In your views, in order to create links make use of the Rails methods `link_to`,`button_to`, and `link_to image_tag` like these examples:

```ruby
link_to "Improve Your Ruby Skills", "/ruby-book" %>
button_to "Improve Your Ruby Skills", "/ruby-book" %>
link_to image_tag("path/to/image.png"), "http://www.mysite.com/"
```

That way, you will be able to navigate your project 💪

File: app/views/share/_post.html.erb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    <li class="card mb-3 w-75 mt-4" >
+      <div class="col-md-12">
+      <div class="card-body">
+      <h4 class="card-title">Post #<%= post.id %></h4><br>
+      <h3 class="card-title"><%= post.title %></h3>
+      <p class="card-text" ><%= post.text %></p>
+       <p class="card-text text-end">
+        <small class="text-muted">
+          comments: <%= !post.comments_counter ? 0 :  post.comments_counter %>,
+          likes: <%= !post.likes_counter ? 0 :  post.likes_counter %>
```
Comment (Meltrust) [inline app/views/share/_post.html.erb:11]: - [ ] I was unable to create a `like` in the console in order to check the views.  Tried several times:

image

Please, double-check `likes` can be created 💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @Mithi-code,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- Nice functionality and styling✔️ 

image


You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by descholar-ceo — APPROVED** [review]
Hi @Mithi-code , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**